examples/flaskr/flaskr/auth.py

```diff
@@ -0,0 +1,121 @@
+import functools
+
+from flask import Blueprint
+from flask import flash
+from flask import g
+from flask import redirect
+from flask import render_template
+from flask import request
+from flask import session
+from flask import url_for
+from sqlalchemy.ext.hybrid import hybrid_property
+from werkzeug.security import check_password_hash
+from werkzeug.security import generate_password_hash
+
+from flaskr import db
+
+
+class User(db.Model):
```

Comment:
do we really want to mix models and blueprints in the same file?

Reply:
You mean make the blueprint a package with `views` and `models` instead?